swig/build-utils.shinc

```diff
@@ -47,6 +47,27 @@ macosx_env() {
   export OPENSSL_ROOT="${SRC}/openssl"
 }
 
+ios_env() {
+    abort_if_var_unset "HOME" ${HOME}
+    export SRC="${HOME}/src"
+    export OPENSSL_ROOT="${SRC}/openssl"
+    export IOS_SDK_VERSION="14.0"
+    export IOS_ARCH="arm64"
+    export IOS_PLATFORM="iphoneos"
+    export IOS_COMPILER="iphoneos"
+
+    XCODE_SDK_PATH=$(xcode-select -p)/Platforms/${IOS_PLATFORM}.platform/Developer/SDKs/${IOS_PLATFORM}${IOS_SDK_VERSION}.sdk
+    export IOS_SYSROOT="${XCODE_SDK_PATH}"
```

Comment:
hmm

Reply:
If you're not going to use XCODE_SDK_PATH elsewhere, why not just:
`export IOS_SYSROOT=$(xcrun --sdk macosx --show-sdk-path)`

Reply:
no need for a XCODE_SDK_PATH variable, just do
```
export IOS_SYSROOT=$(xcrun --sdk iphoneos --show-sdk-path)
```